### DISCUSSION TITLE
How to chain two projects under one bot?

### DISCUSSION THREAD

**Discussion by blshkv** [aiogram/aiogram#1022] [Q&A]
Hi guys,

I want to have 1 bot in my chat room and connect few other projects (written in different languages) in chain to handle webhook events. Each project accepts a different set of commands.
Is any way to create such setup? Can't figure out what would work the best.

Thanks!

**Comment by JrooTJunior** [comment]
Seems like you need an gateway for all other services

**Reply by blshkv** [reply]
hm.. I checked a webhook gateway (https://github.com/HeCorr/telegram-bot-gateway).
It seems like it expects two registered bots with two registered webhook URLs. But I can't  do that, since I have 1 bot only.

I think what I really want is to pass unprocessed dispatcher event to a next bot with its own local URL.
